### ISSUE TITLE
 contrast loss function uses InfoNCE

### ISSUE THREAD

**Issue by ytaochen** [cong-wu/SCD-Net#3]
I didn't find that the contrast loss function uses InfoNCE in the code you provided. Can you explain it?

**Comment by cong-wu** [comment]
Hi, you can find the loss function from ./scd/builder.py and ./pretraining.py, you can also find the similar implementation from https://github.com/HuiGuanLab/HiCo.

**Comment by ytaochen** [comment]
Ok, thanks for your reply



---Original---
From: "Cong ***@***.***&gt;
Date: Tue, Jan 7, 2025 19:27 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [cong-wu/SCD-Net]  contrast loss function uses InfoNCE (Issue #3)




 
Hi, you can find the loss function from ./scd/builder.py and ./pretraining.py, you can also find the similar implementation from https://github.com/HuiGuanLab/HiCo.
